pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/omregning/ufoeretrygd/Ufoeretrygd.kt

```diff
@@ -149,19 +149,39 @@ object Ufoeretrygd {
         override fun OutlineOnlyScope<LangBokmalNynorsk, Unit>.template() {
             paragraph {
                 text(
-                    Bokmal to "Du er tidligere innvilget rettighet som ung ufør i uføretrygden din. Denne rettigheten gir deg høyere utbetaling fra og med den måneden du fyller 20 år.",
-                    Nynorsk to "Du har tidlegare fått innvilga rett som ung ufør i uføretrygda di. Denne retten gir deg høgare utbetaling frå og med den månaden du fyller 20 år.",
+                    Bokmal to "Du har tidligere fått innvilget uføretrygd som ung ufør. Fra og med den måneden du fyller 20 år har du rett til høyre minsteytelse.",
+                    Nynorsk to "Du har tidlegare innvilga rett som ung ufør i uføretrygda di. Frå og med månaden du fyller 20 år har du rett til høgare minsteyting.",
                 )
             }
+            paragraph {
```

Comment:
Does this text belong here in this phrase? if so, can you include the old reference to this text in the comment above ?     TBU3008, TBU3009, TBU3010, ???